### ISSUE TITLE
Apiref

### ISSUE THREAD

**Issue by Mattis44** [Mattis44/fast-react-calendar#12]
Devs should be able to use Apiref passed as a hook `useApiRef` to get an interface of all states/methods available to interact with `FastCalendar` component.

```ts
const apiRef = useApiRef();

<FastCalendar
   apiRef={apiRef}
/>
```
methods (not finished):

- apiRef.current.setMonth(...)
...
